jwst/pipeline/tweakreg.cfg

```diff
@@ -5,8 +5,6 @@ save_catalogs = False
 catalog_format = 'ecsv'
 kernel_fwhm = 2.5
 snr_threshold = 10.
-brightest = None
```

Comment:
@jdavies-st Make this a 100 default... This should keep top 100 sources for alignment and 100 should be enough to get a good alignment. This will help eliminate low SNR or fake sources and thus make the algorithm more robust against low SNR detection thresholds.